staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -930,10 +945,9 @@ func (e *Store) Delete(ctx genericapirequest.Context, name string, options *meta
 	if err != nil {
 		return nil, false, err
 	}
-
 	obj := e.NewFunc()
 	if err := e.Storage.Get(ctx, key, "", obj, false); err != nil {
-		return nil, false, storeerr.InterpretDeleteError(err, e.QualifiedResource, name)
+		return nil, false, storeerr.InterpretDeleteError(err, e.qualifiedResourceFromContext(ctx), name)
```

Comment:
Same comment about storing.

> Be consistent with the rest of your changes and store `qualifiedResource := e.qualifiedResourceFromContext(ctx)` since it used more than once in this func.